release-notes/opensearch-ml-common.release-notes-2.5.0.0.md

```diff
@@ -0,0 +1,37 @@
+## Version 2.5.0.0 Release Notes
+
+Compatible with OpenSearch 2.5.0
+
+### Features
+* support uploading prebuilt model ([#655](https://github.com/opensearch-project/ml-commons/pull/655))
+* disable prebuilt model ([#682](https://github.com/opensearch-project/ml-commons/pull/682))
```

Comment:
These two lines seems confusing if user read this release notes as the prebuilt model feature is not in 2.5 release. How about just remove these two PRs ?

Reply:
Nice Catch, that's also what I am thinking about